README.md

```diff
@@ -5,9 +5,11 @@ At this point all it can do is convert `UNSMRY` files into the MessagePack forma
 ## Building
 
 - Install Rust and Cargo. Refer to https://rustup.rs/ for rust installation;
+- Default to using a `nightly` toolchain via `rustup default nightly`.
```

Comment:
I prefer `rustup override set nightly` per crate.

Reply:
I was trying to remember how to do this. I actually prefer that too. :)

Reply:
Added a commit for this specifically.